app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def comments_posts_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#comments_post_counter` method every time a new comment has been saved to the database, then make the `comments_post_counter` method private